### DISCUSSION TITLE
Outpost reports the wrong number of owned items

### DISCUSSION THREAD

**Discussion by tjprescott** [FakeFishGames/Barotrauma#13854] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/5723682/58dabd8a-54f5-4cab-8ad6-9c8c6933cc5e)

Here's an example. It says I own one harpoon gun and no harpoons, but I have over a dozen harpoons on my sub. This causes me to buy a lot more items than I need because I think I've completely run out, but in fact I haven't!  This seem to be a recent development. 

### Reproduction steps

For me this just happens regularly now in single player campaign. My items are stored in the sub on containers but the merchant "owned" values are all off.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13826
